holoviews/core/data/__init__.py

```diff
@@ -15,13 +15,23 @@
 from .grid import GridInterface
 from .ndelement import NdElementInterface
 
+datatypes = ['array', 'dictionary', 'grid', 'ndelement']
+
```

Comment:
Is this because you are considering setting datatypes with ``hv.core.data.datatypes = [ 'dictionary']`` for example? Makes sense, for convenience but I would like to know if there was any another reason for moving it off the class itself.